src/commands/deploy.rs

```diff
@@ -656,7 +658,17 @@ async fn check_healthz(base_url: &Url) -> Result<()> {
 
 const READINESS_POLL_INTERVAL_SECS: u64 = 2;
 
-async fn wait_for_ready(app_base_url: &Url, bindle_version: &str, readiness_timeout_secs: u16) {
+enum Destination {
+    Cloud(String),
+    Platform,
+}
+
+async fn wait_for_ready(
+    app_base_url: &Url,
+    bindle_version: &str,
+    readiness_timeout_secs: u16,
+    destination: Destination,
```

Comment:
I probably would have gone with:

```suggestion
    dashboard_url: Option<&str>,
```

(but this is fine too)

Reply:
I prefer the enum because when I read the call site I see `Destination::Platform` instead of the rather uninformative `None`.  But it's subjective for sure.

Reply:
For better or worse Rust has decided not to use labeled function arguments :shrug:. It would be really helpful if Github could inline rust-analyzer hints into PR diffs.